packages/falso/src/tests/core/object-with-id-is-unique.spec.ts

```diff
@@ -0,0 +1,46 @@
+import { objectWithIdIsUnique } from '../../lib/core/unique-validators';
+import { randFullName, randNumber } from '@ngneat/falso';
+
+describe('objectWithIdIsUnique', () => {
+  let objects: any[];
+  let object1: any;
+  let object2: any;
+  let object3: any;
+
+  beforeEach(() => {
+    const name = randFullName();
+    const age = randNumber({ min: 10, max: 90 });
+    object1 = {
+      id: '1',
+      name,
+      age,
+    };
+    object2 = {
+      id: '2',
+      name,
+      age,
+    };
+    object3 = {
+      id: '3',
+      name,
+      age,
+    };
+    objects = [object1, object2];
+  });
+
+  describe('object id is unique', () => {
+    it('should return true', () => {
```

Comment:
I agree with Netanel's previous comment about the naming.
Maybe something like:
`it(Should pass when object ids are unique)` instead of the describe. This goes to all the specs I've seen in the recent files.
@NetanelBasal thoughts?

Reply:
Agree